src/App.js

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.scss';
-
 import todos from './api/todos';
 import users from './api/users';
 
+const prepearedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
 function App() {
   return (
-    <div className="App">
+    <div className="container">
       <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <TodoList prepearedTodos={prepearedTodos} />
```

Comment:
```suggestion
      <TodoList todos={prepearedTodos} />
```
немає потреби називати їх одинаково, і мені здається тут логічніше писати просто todos